src/extractor/extractor.cpp

```diff
@@ -673,25 +673,24 @@ void Extractor::WriteIntersectionClassificationData(
     util::RangeTable<> bearing_class_range_table(bearing_counts);
     file_out_stream << bearing_class_range_table;
 
-    file_out_stream << total_bearings;
+    file_out_stream.write( reinterpret_cast<const char*>(&total_bearings), sizeof(total_bearings));
     for (const auto &bearing_class : bearing_classes)
     {
         const auto &bearings = bearing_class.getAvailableBearings();
         file_out_stream.write(reinterpret_cast<const char *>(&bearings[0]),
                               sizeof(bearings[0]) * bearings.size());
     }
 
-    // FIXME
```

Comment:
@daniel-j-h You said something about removing this here. Can you elaborate what is required to fix this, is actin required here?